### ISSUE TITLE
General Linux 64 Build Instructions (or Script), or a General Binary

### ISSUE THREAD

**Issue by Mcpower03** [minbrowser/min#1308]
**Can you describe the problem you're facing?**
 I run Solus Linux, and there is no Min package available

**What solution would you like?**
Just instructions, or an additional script to build for generic Linux 64. A dedicated Solus package would be nice, but that seems like a lot to ask. A generic Linux binary would also work

**Are there any alternatives you've considered?**
I would have tried building it myself, but there is only scripts for Ubuntu and Fedora/Red Hat

**Comment by PalmerAL** [comment]
If you run any of the linux packaging commands, you'll also get a `min-linux-*` folder with a binary and some resource files; I'd imagine that's what you want.

**Comment by Mcpower03** [comment]
Is it a 'universal' binary, or a debian or red hat (depending on the command) binary?

**Comment by PalmerAL** [comment]
I think it's universal? It's whatever Electron provides by default.

It's probably best to just run it and see if it works.

**Comment by Mcpower03** [comment]
Running the buildRedhat command produces this error
![Screenshot_20201030_204319](https://user-images.githubusercontent.com/54217969/97767134-94737580-1af0-11eb-8cd0-6d33be7d9b4e.png)
With this being the log file
[2020-10-31T00_42_14_785Z-debug.log](https://github.com/minbrowser/min/files/5468265/2020-10-31T00_42_14_785Z-debug.log)

**Comment by PalmerAL** [comment]
Yeah, the script won't finish running, but before it creates the package, it should create the folder I mentioned above. So let the script fail, and then look for the executable in that folder and try to run it.

**Comment by Mcpower03** [comment]
That seems to have worked, what would you suggest for some desktop integration (so I'm not just going into a file browser and running a binary every time I want to browse the web). I have a .desktop file, but its just kind of a template one from firefox filled in with the path to the binary and icon. Also, where can I find a high res icon or a svg version of the icon

**Comment by Syndamia** [comment]
> Also, where can I find a high res icon or a svg version of the icon

If you have a mac, you can export [the icon source files](https://github.com/minbrowser/min/tree/master/icons/source) to whatever format you want. 

If not, [this](https://user-images.githubusercontent.com/46843671/97772488-5d34b100-1b50-11eb-8749-2e646271275e.png) is the best there is.

**Comment by Mcpower03** [comment]
Great thanks, unfortunately I don't have a mac, and from a little research I wouldn't be able to convert it to svg anyway, but thanks for the help! Your welcome to close this issue, or you can leave it up so other people know what to do if they have the same problem
